Examples/FriendlyFlix/README.md

```diff
@@ -0,0 +1,73 @@
+# FriendlyFlix
+
+## Introduction
+
+This quickstart is a movie tracker app to demonstrate the use of Firebase Data Connect
+ with a Cloud SQL database.
+
+For more information about Firebase Data Connect visit [the docs](https://firebase.google.com/docs/data-connect/).
+
+## Getting Started
+
+Follow these steps to get up and running with Firebase Data Connect. For more detailed instructions,
+check out the [official documentation](https://firebase.google.com/docs/data-connect/quickstart).
+
+### 0. Prerequisites
+- Latest version of [Xcode](https://developer.apple.com/xcode/)
+- Latest version of [Visual Studio Code](https://code.visualstudio.com/)
+- The [Firebase Data Connect VS Code Extension](https://marketplace.visualstudio.com/items?itemName=GoogleCloudTools.firebase-dataconnect-vscode)
```

Comment:
I would instead link to the prereqs from the Firebase docs site

Reply:
Tyler suggested keeping the README as self-contained as possible, so I'd rather include this here.